### DISCUSSION TITLE
rAudio 1 • 20220116 - No more thumbnails in playlist window

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#767] [Fixed Issues]
Available thumbnails (thumb.jpg) are no more displayed in the playlist windows.

**Comment by ceejayemm** [comment]
@rern 

rAudio-1 seems to be looking for thumbs.jpg in the wrong folder (from journalctl -xe)

Jan 17 09:49:22 rAudio-1 nginx[351]: 2022/01/17 09:49:22 [error] 351#351: *36 open() "/srv/http/mnt/MPD/NAS/NAS_Music/Pink Floyd/thumb.jpg" failed (2: No such file or directory), client>
Jan 17 09:49:22 rAudio-1 nginx[351]: 2022/01/17 09:49:22 [error] 351#351: *35 open() "/srv/http/mnt/MPD/NAS/NAS_Music/Red Box/thumb.jpg" failed (2: No such file or directory), client: 1>
lines 2637-2660/2660 (END)

Should be, respectively;

/srv/http/mnt/MPD/NAS/NAS_Music/Pink Floyd/Wish You Were Here/thumb.jpg

and 

/srv/http/mnt/MPD/NAS/NAS_Music/Red Box/Plenty/thumb.jpg

In all cases it seems to be missing the album name in the search for the thumbs.jpg file

Chris

**Reply by klassikmann** [reply]
@ceejayemm 
Good Analysis!

**Reply by rern** [reply]
@klassikmann, @ceejayemm,

Please try the UPDATE branch.

**Reply by klassikmann** [reply]
> @klassikmann, @ceejayemm,
> 
> Please try the UPDATE branch.
Fixed.

**Reply by ceejayemm** [reply]
Couldn't get to test this straightaway but I see you have incorporated the fix into 2022017. I applied this update and all seems to be well now.  The errors reported thru journalctl -xe all seem to have gone as well.

Thanks

Chris
